src/Interpreters/Cache/LRUFileCachePriority.cpp

```diff
@@ -157,6 +164,7 @@ void LRUFileCachePriority::LRUFileCacheIterator::annul()
 {
     updateSize(-queue_iter->size);
     chassert(queue_iter->size == 0);
+    --cache_priority->current_elements_num;
```

Comment:
why don't do it inside `updateSize()`. In this case we don't log `FilesystemCacheElements` event